bin/setenv-docker-customize.sh

```diff
@@ -608,25 +608,31 @@ else
     _ADDON_MGR_OPTION_PATCHES_CATALOG="--catalog=${EXO_PATCHES_CATALOG_URL}"
   fi
 
-  # add-ons removal
-  if [ -z "${EXO_ADDONS_REMOVE_LIST:-}" ]; then
-    echo "# no add-on to uninstall from EXO_ADDONS_REMOVE_LIST environment variable."
+  if [ -f /opt/exo/_done.addons_removal ]; then
+    echo "INFO: add-ons removal already done! skipping this step."
   else
-    echo "# uninstalling default add-ons from EXO_ADDONS_REMOVE_LIST environment variable:"
-    echo ${EXO_ADDONS_REMOVE_LIST} | tr ',' '\n' | while read _addon ; do
-      if [ -n "${_addon}" ]; then
-        # Uninstall addon
-        ${EXO_APP_DIR}/addon uninstall ${_addon}
-        if [ $? != 0 ]; then
-          echo "[ERROR] Problem during add-on [${_addon}] uninstall."
-          exit 1
```

Comment:
The problem is related to the installation phase. If this fails, the exo container is restarted and attempts to uninstall the already done add-ons, resulting in another boot problem. This PR will prevent redoing the uninstall phase to allow exo to focus directly on the install phase (e.g. network problem, it will restart indefinitely until the install phase succeeds)